zulipterminal/ui_tools/boxes.py

```diff
@@ -62,8 +63,13 @@ def private_box_view(self, button: Any=None, email: str='',
         ]
         self.focus_position = 1
 
-    def stream_box_view(self, caption: str='', title: str='') -> None:
+    def stream_box_view(self, stream_id: int, caption: str='', title: str='',
+                        ) -> None:
         self.set_editor_mode()
+        subscribers = deepcopy(self.model.get_subscribers_in_stream(stream_id))
```

Comment:
Why not return a new list in the method, and skip the need for the deepcopy? Alternatively, we're only using it here so far, so we could change it to be `get_other_subscribers_in_stream` and incorporate the extra code below?

Reply:
Updated to return new list, but this method can be used for displaying subscriber counts

Reply:
I hope returning a new list is the same as using deepcopy, so I would just be moving the deepcopy to the model?

Reply:
OK, modified it to `get_other_subscribers_in_stream()` and used the `stream_dict` directly for subscribers counts. Removed the test because I don't see a point in writing an entire test just for a simple `assert_called_once_with()`. This can added when the whole method is tested.